### PR TITLE
Feature/je 237 error handling

### PR DESCRIPTION
**Pull request by tuj**
https://jira.itkdev.dk/browse/JE-239

* Added required to invoice entry form inputs.
* Added checks that invoice entries are okay, before allowing an invoice to be recorded.
* Added custom fetch adapter to get exception messages.
* Added tmpfs to docker.
* Changed csv export encoding to Windows-1252.

### CODE REVIEW

File: bundles/Billing/Controller/ApiController.php [hunk 1]
```diff
@@ -241,7 +242,13 @@ public function getProjectWorklogs(BillingService $billingService, $projectId)
      */
     public function recordInvoice(BillingService $billingService, $invoiceId)
     {
-        return new JsonResponse($billingService->recordInvoice($invoiceId));
+        try {
+            $invoice = $billingService->recordInvoice($invoiceId);
+
+            return new JsonResponse($invoice);
+        } catch (Exception $e) {
```
Comment (rimi-itk) [inline bundles/Billing/Controller/ApiController.php]: This is a very general exception to catch. Do we risk hiding exceptions that are not directly related to the actual service (api) call?

File: bundles/Billing/Service/BillingService.php [hunk 2]
```diff
@@ -776,21 +833,32 @@ public function exportInvoicesToSpreadsheet(array $invoiceIds)
...
                 $sheet->setCellValueByColumnAndRow(1, $row, 'L');
                 // B. "Materiale (vare)nr.
-                $sheet->setCellValueByColumnAndRow(2, $row, str_pad($invoiceEntry->getMaterialNumber(), 18, '0', STR_PAD_LEFT));
+                $sheet->setCellValueByColumnAndRow(2, $row, str_pad($materialNumber, 18, '0', STR_PAD_LEFT));
                 // C. "Beskrivelse"
-                $sheet->setCellValueByColumnAndRow(3, $row, $invoiceEntry->getProduct());
+                $sheet->setCellValueByColumnAndRow(3, $row, $product);
                 // D. "Ordremængde"
-                $sheet->setCellValueByColumnAndRow(4, $row, number_format($invoiceEntry->getAmount(), 3, ',', ''));
+                $sheet->setCellValueByColumnAndRow(4, $row, number_format($amount, 3, ',', ''));
```
Comment (rimi-itk) [inline bundles/Billing/Service/BillingService.php:856]: Maybe use ”Number formats”: https://phpspreadsheet.readthedocs.io/en/latest/topics/recipes/#number-formats?

**Review by rimi-itk — CHANGES_REQUESTED** [review]

File: bundles/Billing/Service/BillingService.php (hunk 2 above)
Comment (tuj) [reply bundles/Billing/Service/BillingService.php:856]: I will leave this as is atm.

File: bundles/Billing/Controller/ApiController.php (hunk 1 above)
Comment (tuj) [reply bundles/Billing/Controller/ApiController.php]: Change to custom exception

**Review by rimi-itk — APPROVED** [review]
